### PR TITLE
Added the code for Fcfs scheduling algorithm

### PR DESCRIPTION
**Pull request by MihSawant**
### **Describe your change:**

-  [  ] Add an algorithm?
   Added the Fcfs cpu scheduling algorithm
- [ ] Fix a bug or typo in an existing algorithm?
     Implemented the algorithm
- [ ] Documentation change?
    Nothing changed in documentation

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ ] All new Java files are placed inside an existing directory.
- [ ] All filenames are in all uppercase characters with no spaces or dashes.
- [ ] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains. Issue #3291 which was assigned to me.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,6 +65,14 @@
...
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>16</source>
+                    <target>16</target>
+                </configuration>
+            </plugin>
```
Comment (siriak) [inline pom.xml:75]: It's already present above, why do you need to duplicate it?

File: src/main/java/com/thealgorithms/others/Fcfs.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    }
+
+
+    public void printTable(){
+        System.out.printf("%s\t%s\t%s\t%s%n", "AT","BT","WT","TAT");
+        for(int i = 0; i < n; i++){
+            System.out.printf("%d\t%d\t%d\t%d%n", arrivalTime[i], burstTime[i], waitingTime[i], turnAroundTime[i]);
+        }
+    }
```
Comment (siriak) [inline src/main/java/com/thealgorithms/others/Fcfs.java:34]: Please remove this, it's redundant

File: src/main/java/com/thealgorithms/others/Fcfs.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    public double getAvgWaitingTime(){
```
Comment (siriak) [inline src/main/java/com/thealgorithms/others/Fcfs.java:35]:
```suggestion
    public double getAverageWaitingTime(){
```

File: src/main/java/com/thealgorithms/others/Fcfs.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        double sum = 0;
+        double avg = 0;
+        for(int i = 0; i < n; i++){
+            sum += waitingTime[i];
+        }
+        avg = sum / n;
+        return avg;
+    }
+
+    public double getAvgTurnAroundTime(){
```
Comment (siriak) [inline src/main/java/com/thealgorithms/others/Fcfs.java:45]:
```suggestion
    public double getAverageTurnaroundTime(){
```

File: src/main/java/com/thealgorithms/others/Fcfs.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        double sum = 0;
+        double avg = 0;
+        for(int i = 0; i <n; i++){
+            sum += turnAroundTime[i];
+        }
+        avg = sum / n;
+        return avg;
+    }
+
+    public static void main(String... args){
```
Comment (siriak) [inline src/main/java/com/thealgorithms/others/Fcfs.java:55]: Please put code in appropriately named methods, add JUnit tests and remove Hamming distance from this PR

**Review by siriak — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (MihSawant) [reply pom.xml:75]: Sorry Sir, I am new to github, doing PR's, can you please help me out with how can i update only the code which is latest change ? And not the entire thing like this happened...

Comment (siriak) [reply pom.xml:72]: See `git add` command if that's what you're talking about

**Pull request closed without merging**